src/interfaces/director-interface.ts

```diff
@@ -1,4 +1,26 @@
+import { AddressIF, EmptyAddress, OfficerIF, EmptyOfficer } from '@/interfaces'
+import { Actions } from '@/enums'
+
 export interface DirectorIF {
-  actions?: string[];
-  isDirectorActionable?: boolean;
+  id: number
+  officer: OfficerIF
+  deliveryAddress: AddressIF
+  mailingAddress?: AddressIF
+  appointmentDate: string
+  cessationDate: string
+  cessationDateTemp?: string
+  isFeeApplied?: boolean
+  isDirectorActionable?: boolean
+  actions: Array<Actions>
+}
+
+export const EmptyDirector: DirectorIF = {
+  id: null,
+  officer: { ...EmptyOfficer },
+  deliveryAddress: { ...EmptyAddress },
```

Comment:
I'm using spread/merge here to handle "deep copy".
(Ie, In cause anyone who uses "EmptyDirector" does a shallow copy, and here I handle the sub-properties being copied. So the result is an all-new object, no object reference copies.)